pangeo_forge_recipes/transforms.py

```diff
@@ -543,9 +525,9 @@ def expand(self, references: beam.PCollection) -> beam.PCollection[zarr.storage.
             | CombineReferences(
                 concat_dims=self.concat_dims,
                 identical_dims=self.identical_dims,
-                target_options=self.target_options,
-                remote_options=self.remote_options,
-                remote_protocol=self.remote_protocol,
+                target_options=self.target_root.get_fsspec_credential_kwargs(),
+                remote_options=self.target_root.get_fsspec_credential_kwargs(),
+                remote_protocol=self.target_root.get_fsspec_remote_protocol(),
```

Comment:
again: general idea is that instead of doing something gross like unpacking auth creds from an `FSSpec` instance we just use `config.Config.target_storage` from https://github.com/pangeo-forge/pangeo-forge-recipes/pull/652 here